src/python/dxpy/utils/describe.py

```diff
@@ -918,6 +918,14 @@ def print_execution_desc(desc):
         if field not in recognized_fields:
             print_json_field(field, desc[field])
 
+def print_currency(value, meta):
+    prefix = '-' if value < 0 else ''
+    prefix += meta['symbol'] if meta['symbolPosition'] == 'left' else ''
+    suffix = f' {meta["symbol"]}' if meta['symbolPosition'] == 'right' else ''
```

Comment:
Are f strings supported in Python 2.7? If not, we probably need to change to `str.format()` here for compatibility.  

We have not yet officially deprecated Python 2.7 support, still building for the 16.04 job execution environment and tarball published to the Downloads page in our docs.

Reply:
I see, I assumed we've moved over to python 3+ based on the commend in README.md (I think it states April 2021 as end of support). I can rewrite to 2.7 np